src/Modules/CalcOffence.lua

```diff
@@ -3210,7 +3210,11 @@ function calcs.offence(env, actor, activeSkill)
 			local durationMod = m_max(calcLib.mod(skillModList, dotCfg, "EnemyIgniteDuration", "SkillAndDamagingAilmentDuration") * calcLib.mod(enemyDB, nil, "SelfIgniteDuration"), 0)
 			globalOutput.IgniteDuration = durationBase * durationMod / rateMod * debuffDurationMult
 			globalOutput.IgniteDuration = globalOutput.IgniteDuration > data.misc.IgniteMinDuration and globalOutput.IgniteDuration or 0
-			local igniteStacks = (globalOutput.IgniteDuration / output.Time) / maxStacks
```

Comment:
you still need to define it as local, either by adding local below, or the better way to do this would be 
local igniteStacks = 1
if not skillData.triggeredOnDeath then
	igniteStacks = (globalOutput.IgniteDuration / output.Time) / maxStacks
end